CommonVoice-Data/framabook.py

```diff
@@ -0,0 +1,253 @@
+import os
+import re
+import argparse
+import unicodedata
+
+import ebooklib
+from ebooklib import epub
+
+from bs4 import BeautifulSoup
+import spacy
+
+from collections import Counter
+from utils import splitIntoWords, filter_numbers, maybe_normalize, extract_sentences, check_output_dir, set_custom_boundaries
+
+
+# keep only some epub items whose names match the regex
+REGEX_CHAPITRE = r'.*ch(apitre)?\-?[0-9]+\.xhtml$'
+
+# mapping for abbr tags
+ABBRS_MAPPING = {
```

Comment:
J'aime bien cette liste, tu pourrais peut-être la partager plus largement sur le repo et / ou avec Corpora Creator après :)

Reply:
Ok ! C'est noté :)
Petite note : comme les magnifiques bénévoles de framabook font du beau boulot sur les epubs, j'ai établi la liste des clés de `ABBRS_MAPPING` avec les tags `<abbr>` des epubs (du coup on est sûr de ne pas avoir d'autres choses que des abréviations).

Reply:
Ah c'est cool, parce que si tu t'embêtes, côté Projet Gutenberg c'est pas la même limonade, y'a beaucoup de déchets :D